packages/types/src/index.ts

```diff
@@ -307,6 +307,14 @@ export type Leg = {
   mode: string;
   pathway: boolean;
   pickupBookingInfo?: FlexPickupBookingInfo;
+  rideHailingEstimate?: {
```

Comment:
Can you add `productName?: string` to this type so that it matches what exists in the API? That way we don't need to do an update to types if we ever want that property. 
<img width="408" alt="image" src="https://github.com/opentripplanner/otp-ui/assets/92878779/66d8059e-5e3a-433c-a7f2-774beeaac3f8">

Reply:
Good catch thanks